name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/SBrokerPDFExtractor.java

```diff
@@ -1511,6 +1511,42 @@ private void addAccountStatementTransaction()
 
                             return item;
                         }));
+
+        Block taxReturnBlock_Format01 = new Block("^.* [\\-|\\+|\\s][\\.,\\d]+$");
+        taxReturnBlock_Format01.setMaxSize(3);
+        type.addBlock(taxReturnBlock_Format01);
+        taxReturnBlock_Format01.set(new Transaction<AccountTransaction>()
+
+                        .subject(() -> {
+                            AccountTransaction accountTransaction = new AccountTransaction();
+                            accountTransaction.setType(AccountTransaction.Type.TAX_REFUND);
+                            return accountTransaction;
+                        })
+
+                        .section("type", "amount", "date", "note").optional() //
+                        .documentContext("currency") //
+                        .match("^.* (?<type>[\\-|\\+|\\s])(?<amount>[\\.,\\d]+)$") //
```

Comment:
"`<type>`" is not used.... I think you need to change from "tax refund" to "tax" 💯. 

```
   // Is type --> "-" change from TAX_REFUND to TAXES
   if ("-".equals(trim(v.get("type"))))
      t.setType(PortfolioTransaction.Type.TAXES);
```